fhservices.cfg.j2

```diff
@@ -121,6 +121,7 @@ define service {
 
 define service {
        service_description Storage::Pod::Disk Usage
+       # REVIEW: is this being applied to every pod/container? How does it work?
```

Comment:
What exactly does the pod discovery? Could you send a link to it?
We need to ensure this works for multi-container pods.